bili.config.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
   vue: {
     css: false,
     template: {
-      isProduction: true,
+      isProduction: false,
```

Comment:
Why we turn `false` this part?

Reply:
Ahh, I was finishing up another project while packing for a conference. It looks like a few mistakes slipped through. I’ll review these, squash the commits and resubmit.